### DISCUSSION TITLE
Manual/class-EditorManager

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#8] [Notes]
# Manual/class-EditorManager

https://docs.unity3d.com/Manual/class-EditorManager.html

**Comment by vertxxyz** [comment]
Mixed mode serialisation will keep assets in the formats they have been serialised in. New assets are likely serialised in binary. It's a temporary setting intended for upgrading projects, so preferably choose either Text or Binary serialisation. You cannot set the mode on a per-asset basis.
